src/test/test_parseconf.sh

```diff
@@ -125,12 +128,30 @@ for dir in "${EXAMPLEDIR}"/*; do
         CMDLINE=""
     fi
 
-    if test -f "./expected"; then
-        if test -f "./error"; then
-            echo "FAIL: Found both ${dir}/expected and ${dir}/error."
-            echo "(Only one of these files should exist.)"
-            exit $EXITCODE
+    # If tor has some modules disabled, search for a custom result file for
+    # the disabled modules
+    for prefix in "$TOR_MODULES_DISABLED" ""; do
+
+        if test -f "./${prefix}expected"; then
```

Comment:
I am a little concerned about having relay_expected  mean "expected when NOT built a asrelay."  I would have expected something more like "norelay_expected" or expected_norelay.

Reply:
Changed to expected_no_dirauth in 3d38436d94e32cdf7e8157c3447bc70ee8328357 and d3f798096f725b1ebfe495af6e9c6c4c9a0460ce

Also added documentation for the module-based results files.